playwright/navigation.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { expect, test } from '@playwright/test';
+
+test('Navigate and check content', async ({ page }) => {
+  await page.goto('/');
+  await expect(page.getByTestId('myHero')).toBeVisible();
+  await expect(page.getByTestId('whoAmI')).toBeVisible();
+  await expect(page.getByTestId('myWork')).toBeVisible();
+  await expect(page.getByTestId('footer')).toBeVisible();
+
+  await page.getByRole('link', { name: 'My work' }).click();
+  await page.waitForURL('/#portfolio');
+  await expect(page.getByTestId('myWork')).toBeInViewport();
+
+  await page.getByRole('link', { name: 'Contact' }).click();
+  await page.waitForURL('/#about');
+  await expect(page.getByTestId('whoAmI')).toBeInViewport();
```

Comment:
these are 3 different strings here, I guess that doesn't make so much sense?

Reply:
As mentioned in this [comment](https://github.com/upleveled/portfolio-cms/pull/112#discussion_r1311689909), there is no `contact` section. I changed it to "About me"
commit [e228460](https://github.com/upleveled/portfolio-cms/pull/112/commits/e22846068bc93a4f8055076a6634814b277d3b1b)